### DISCUSSION TITLE
Python PoC

### DISCUSSION THREAD

**Discussion by RobMarion** [omnibor/spec#45] [General]
Are we still looking for a Python PoC? I could give it a stab but would like to know what the guidelines & expectations are.
@AevaOnline @edwarnicke

**Comment by alilleybrinker** [comment]
I believe we are! That said, we'd expressed interest in the possibility of building out different language's implementations on top of the currently-in-draft Rust implementation. I'd be happy to talk about that if you're interested! That said, I think having something working is more important than having something built on Rust.
